docs/index.md

```diff
@@ -0,0 +1,70 @@
+# Iterative Provider
+
+Use the Iterative Provider to launch resource-intensive tasks in popular cloud
+providers with just a simple Terraform file.
+
+## Example Usage
+
+```hcl
+terraform {
+  required_providers {
+    iterative = {
+      source  = "iterative/iterative"
+    }
+  }
+}
+
+provider "iterative" {}
+
+resource "iterative_task" "task" {
+  name  = "example"
+  cloud = "aws"
+
+  script = <<-END
+    #!/bin/bash
+    echo "hello!"
+  END
+}
+```
+
```

Comment:
-> **Note:** See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) for more information.